spec/controllers/self_assessments_controller_spec.rb

```diff
@@ -69,7 +69,15 @@ def valid_session
         post :create, {:self_assessment => {}, :review_id => @review.id}, valid_session
         response.should render_template("new")
       end
+
+      it "throws an error for a blank response" do
+        post :create, {:self_assessment => { response: nil },
+          :review_id => @review.id}, valid_session
```

Comment:
Align the elements of a hash literal if they span more than one line.<br>Use the new Ruby 1.9 hash syntax.<br>Space inside } missing.